### DISCUSSION TITLE
HALMET sample firmware build bombs out

### DISCUSSION THREAD

**Discussion by ESchlotfeldt** [hatlabs/discussions#121] [Q&A] [answered]
Trying to build and upload sample firmware for HALMET. It bombs out with the following message:

<img width="851" alt="Screenshot Halmet firmware build" src="https://github.com/hatlabs/discussions/assets/106944628/0dbab033-88b2-4d29-b4a0-d4379b3c0264">

**Comment by mairas** [comment, marked as answer]
Try deleting the .platformio directory under your home directory. That'll force a full reinstall of the different ESP32 and Arduino frameworks and libraries.

**Reply by ESchlotfeldt** [reply]
Success - thank you!
